assets/js/admin/admin-scripts.js

```diff
@@ -721,7 +672,63 @@ var give_setting_edit = false;
 					});
 				}
 			}).change();
+		},
+
+		saveButtonTriggered: function() {
+			$( 'body' ).on( 'click', '.give-save-button', function () {
```

Comment:
@mehul0810 update this selector also

Reply:
@ravinderk Updated this selector as well.